Makefile

```diff
@@ -181,13 +181,11 @@ OVL_RELOC_FILES := $(shell $(CPP) $(CPPFLAGS) $(SPEC) | grep -o '[^"]*_reloc.o'
 DEP_FILES := $(O_FILES:.o=.asmproc.d) $(OVL_RELOC_FILES:.o=.d)
 
 
-TEXTURE_FILES_PNG := $(foreach dir,$(ASSET_BIN_DIRS),$(wildcard $(dir)/*.png))
-TEXTURE_FILES_JPG := $(foreach dir,$(ASSET_BIN_DIRS),$(wildcard $(dir)/*.jpg))
-TEXTURE_FILES_OUT := $(foreach f,$(TEXTURE_FILES_PNG:.png=.inc.c),build/$f) \
-					 $(foreach f,$(TEXTURE_FILES_JPG:.jpg=.jpg.inc.c),build/$f) \
-
 # create build directories
-$(shell mkdir -p build/baserom build/assets/text $(foreach dir,$(SRC_DIRS) $(UNDECOMPILED_DATA_DIRS) $(ASSET_BIN_DIRS),build/$(dir)))
+stdout := $(shell mkdir -p $(dir $(O_FILES)))
+ifneq ($(stdout),)
+$(error $(stdout))
+endif
```

Comment:
Could `stdout` get a more descriptive/purposed name?

Reply:
so the purpose of these is if somehow something gets printed on stdout
(afaik on any normal system there shouldn't be anything in any case, even assuming the weirdest errors),
it doesn't get parsed as part of the makefile like e.g. an `$(shell echo myvar := myvalue)` line would

and assert the stdout is indeed nothing

I'm open to comments on this because as much as I think it's a proper way to handle things, I'm not a huge fan